src/views/registration-js/attendees/fields/number.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * This template renders a Single Ticket content
+ * composed by Title and Description currently
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/registration-js/attendees/fields/url.php
+ *
+ * @since  TBD
+ * @version TBD
+ *
+ * @see Tribe__Tickets_Plus__Meta__Field__Number
+ *
+ */
+
+$required   = isset( $field->required ) && 'on' === $field->required ? true : false;
+$option_id  = "tribe-tickets-meta_{$field->slug}_{$ticket->ID}{{data.attendee_id}}";
+$field      = (array) $field;
+$field_name = 'tribe-tickets-meta[' . $ticket->ID . '][{{data.attendee_id}}][' . esc_attr( $field['slug'] ) . ']';
+$disabled   = false;
+$classes    = [ 'tribe-common-b1', 'tribe-field', 'tribe-tickets__item__attendee__field__number' ];
+if ( $required ) {
+	$classes[] = 'tribe-tickets-meta-required';
+}
+?>
+<div <?php tribe_classes( $classes ); ?> >
+	<label
+			class="tribe-common-b2--min-medium tribe-tickets-meta-label"
+			for="<?php echo esc_attr( $option_id ); ?>"
+	><?php echo wp_kses_post( $field['label'] ); ?><?php tribe_required_label( $required ); ?></label>
+	<input
+			type="number"
+			min="0" oninput="this.value = Math.abs(this.value)"
```

Comment:
Move the `oninput` into JS, not in the HTML.

Reply:
@paulmskim Do you have a suggestion of where I could port this over to?

Reply:
@Luc45 This would go into the JS file just like you added stuff for the datetime one: https://github.com/moderntribe/event-tickets/pull/1656/files#diff-5833e8be57749d60ff2bc5079f7912fcR551

Reply:
Made suggestion on PR with change I think would work here
```suggestion
			min="0"
```

Reply:
In the JS, when you initialize the JS, you should have a function that binds all the listeners for that page. That would be the place to do this.